src/main/java/frc/robot/subsystems/climber.kt

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration
+import com.ctre.phoenix6.controls.VoltageOut
+import com.ctre.phoenix6.hardware.TalonFX
+import com.ctre.phoenix6.signals.InvertedValue
+import com.ctre.phoenix6.signals.NeutralModeValue
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+import frc.robot.lib.volts
+
+
+object climber : SubsystemBase("climber") {
+
+    private val Motor =
+    TalonFX(16, "*").apply {
+        val config =
+            TalonFXConfiguration().apply {
+                MotorOutput.Inverted = InvertedValue.Clockwise_Positive
```

Comment:
Based on their mounting, this should actually be CounterClockwise_Positive (CCW+)